### DISCUSSION TITLE
dispatch-github-action-workflow-script-action/

### DISCUSSION THREAD

**Discussion by giscus** [estruyf/eliostruyf-comments#627] [Comments]
# dispatch-github-action-workflow-script-action/

Learn how to use the `repository_dispatch` event in GitHub Actions to trigger a workflow with a webhook.

https://www.eliostruyf.com/dispatch-github-action-workflow-script-action/

**Comment by VenkatKandiAmex** [comment]
Thank you, i need to dispatch from a GH repo to another GH Repo which is in an organisation. Is it still possible?

**Reply by estruyf** [reply]
Sure, this functionality is still available. I'm still using it on this website. When I publish or update content, it triggers a build process on another repository, where my Hugo theme is placed.

Are you experiencing something different?
